src/scss/components/navigation/_index.scss

```diff
@@ -42,8 +39,8 @@
 
     &[aria-expanded="true"] {
       .toggle-icon {
-        .icon-transform();
-        transform+_: scaleY(-1);
+        transform: translateY(-50%);
+        transform: scaleY(-1);
```

Comment:
Combine these two properties with the functions in the same order in the value.

```css
transform: translateY(-50%) scaleY(-1);
```

Reply:
Okay done and push the code into git Repository.